parser/parser.go

```diff
@@ -1059,12 +1028,37 @@ func (p *parser) userCall(name string, pos Position) *ast.UserCallExpr {
 			p.commaNewlines()
 		}
 		arg := p.expr()
-		p.processUserCallArg(name, arg, i)
 		args = append(args, arg)
 		i++
 	}
 	p.expect(RPAREN)
-	call := &ast.UserCallExpr{false, -1, name, args} // index is resolved later
-	p.recordUserCall(call, pos)
-	return call
+	return &ast.UserCallExpr{false, ast.WillBeResolvedLater, name, args, pos}
+}
+
+// Record a "multi expression" (comma-separated pseudo-expression
```

Comment:
Yeah, I agree it makes sense to keep the MultiExpr stuff in the parser -- thanks.